process-controller/src/main/java/org/jboss/as/process/logging/ProcessLogger.java

```diff
@@ -289,15 +289,15 @@
      *
      * @return the message.
      */
-    @Message(id = Message.NONE, value = "Keep a copy of the persistent domain configuration even if this host is not the Domain Controller")
+    @Message(id = Message.NONE, value = "Keep a complete copy of the persistent domain configuration even if this host is not the Domain Controller. This parameter will override any ignore-unused-configuration setting in host.xml and attempt to store the complete domain configuration.")
```

Comment:
Let's do that separately though as this is good to go other than a wording tweak here.